homeassistant/components/tesla/__init__.py

```diff
@@ -247,6 +249,7 @@ def device_info(self):
         """Prepare for unload."""
         pass
 
+    @Throttle(timedelta(seconds=MIN_UPDATE_INTERVAL))
     async def async_update(self):
         """Update the state of the device."""
         if self.controller.is_token_refreshed():
```

Comment:
A bit off topic, but is there a guard so that all entities don't save the new token after a refresh?

Reply:
Yes, the `get_tokens` function immediately sets the `is_token_refreshed` flag false.

Reply:
Good! Perhaps add a comment for the future readers about this.